### ISSUE TITLE
High CPU use

### ISSUE THREAD

**Issue by BenHippynet** [savonet/liquidsoap#551]
I'm running Liquidsoap on a single core VPS (Amazon Lightsail). If I run it pulling a single MP3 Icecast stream and transcode it to AAC using FDKAAC the process sits at around 3-4% CPU usage. 

If I add in another transcoder in the same .liq config the CPU usage sits at around 5% but frequently spikes to 100%.

If I add the two transcoders to different .liq files and run them both both processes quickly jump to 100% between them.

I can't figure out why. Am I missing something, or is it possibly because it's a single core CPU so I'll only ever be realistically able to run one transcode on it?

example .liq file:

```
# from an Icecast server or any other source
url = "http://broadcast1.streammachine.co.uk:8084/stream.mp3"
input = mksafe(input.http(url))

# transcoder : AAC 48 kbps
output.icecast(
  %fdkaac(channels=2, samplerate=44100, bitrate=48, afterburner=false, aot="mpeg2_he_aac_v2", transmux="adts", sbr_mode=false),
  mount="/stream.aac",
  host="172.26.10.164", port=8086, password="xxxxx",
  input)

# from an Icecast server or any other source
url = "http://172.26.10.164:8002/stream"
input = mksafe(input.http(url))

# transcoder : AAC 48 kbps
output.icecast(
  %fdkaac(channels=2, samplerate=44100, bitrate=48, afterburner=false, aot="mpeg2_he_aac_v2", transmux="adts", sbr_mode=false),
  mount="/stream.aac",
  host="172.26.10.164", port=8002, password="xxxxxx",
  input)

```

**Comment by mooseh** [comment]
are both the feeds active 100%?

better to make them fallable in that case, you're also defining the same vars for url and input

alos when doing more than 1 thing at a time learn about the "clocks" which enable threading of each task to the CPU

**Comment by toots** [comment]
Hi,

The issue you are describing shouldn't happen. Baring some super weird corner cases, encoding/decoding should scale in a nice linear fashion. 

What version of liquidsoap are you using? The latest dev code (so more recent that `1.3.3` had a bug in `input.http` that could have caused what you are describing.

**Comment by toots** [comment]
ping?

**Comment by toots** [comment]
I'm gonna close it, it seems that this issue was fixed in commit e6159a6. Please reopen if needed.
